### PR TITLE
Expose bindings for lsp_info()

### PR DESCRIPTION
**Pull request by ok300**

### CODE REVIEW

**Review by erdemyerebasmaz — APPROVED** [review]

**Comment by erdemyerebasmaz** [comment]
Adding this API to BreezSDK on sdk-flutter.

**Comment by ok300** [comment]
@dangeross I just added them in 064ed5f3e35b57b9651f56d4c2ffabb17dabd46a

File: libs/sdk-react-native/ios/RNBreezSDK.swift [hunk 1]
```diff
@@ -345,6 +345,16 @@ class RNBreezSDK: RCTEventEmitter {
             rejectErr(err: err, reject: reject)
         }
     }
+
+    @objc(fetchLspInfo:resolver:rejecter:)
```
Comment (erdemyerebasmaz) [inline libs/sdk-react-native/ios/RNBreezSDK.swift:349]: lspInfo

Comment (ok300) [reply libs/sdk-react-native/ios/RNBreezSDK.swift]: Done

File: libs/sdk-react-native/ios/RNBreezSDK.swift [hunk 2]
```diff
@@ -345,6 +345,16 @@ class RNBreezSDK: RCTEventEmitter {
             rejectErr(err: err, reject: reject)
         }
     }
+
+    @objc(lspInfo:resolver:rejecter:)
```
Comment (dangeross) [inline libs/sdk-react-native/ios/RNBreezSDK.swift:349]: Should mirror the RCT_EXTERN_METHOD, so:
```
    @objc(lspInfo:rejecter:)
```

**Review by dangeross — CHANGES_REQUESTED** [review]

File: libs/sdk-react-native/ios/RNBreezSDK.swift (hunk 2 above)
Comment (ok300) [reply libs/sdk-react-native/ios/RNBreezSDK.swift]: Fixed, thanks!

**Review by dangeross — APPROVED** [review]

**Pull request merged**